experimenter/experimenter/nimbus-ui/src/components/Summary/EndExperiment/index.tsx

```diff
@@ -22,7 +22,7 @@ const EndExperiment = ({
           data-testid="end-experiment-start"
           disabled={isLoading}
         >
-          End Experiment
+          End Rollout
```

Comment:
its not about just renaming, you have to see that if the type is rollout like is_rollout is True, then show `End Rollout`, otherwise show `End Experiment`

Reply:
@yashikakhurana I'm currently handling that. This was my initial draft pull request. I want to ensure the tests pass and my changes don't break anything including the mock. I should make a proper PR soon. Thanks for the insight

Reply:
@KramStyles thank you, would highly recommend adding a test, If you will run make check, I am pretty sure there will be broken test

Reply:
@yashikakhurana Oh. Let me start on the tests.

Reply:
I've written the test @yashikakhurana and they've passed.